Makefile

```diff
@@ -77,16 +77,29 @@ end-to-end-test: shared
 	    xvfb-run -a -s '-screen 0 1024x768x24' npm run test -- --verbose
 	sudo docker ps --quiet --all --filter 'ancestor=welder/web-with-coverage' | sudo xargs --no-run-if-empty docker rm -f
 
-cockpit-test: shared
+build-rpm:
+	if [ -n "$$TRAVIS" ]; then \
+	    sudo docker build -f Dockerfile.buildrpm --cache-from welder/buildrpm:latest -t welder/buildrpm:latest .; \
```

Comment:
Does it hurt to have the `--cache-from` on a local build? (I don't see why it would)

Reply:
It does not but the version of Docker on the RHEL builders is older and doesn't support that flag. OTOH we're migrating from Travis (hopefully) and at some point should have docker caching enabled locally so this flag will not be necessary.

Reply:
Ah, that's where it comes from, thanks.